docs/src/_cloud/index.md

```diff
@@ -3,6 +3,7 @@ title: Meltano Cloud
 description: Information about Meltano Cloud Concepts.
 layout: doc
 toc: false
+hidden: true
```

Comment:
Why do we want to hide this? I'm currently having Wil link to the Deployments / Projects section from the main dashboard page.

Reply:
So @tayloramurphy with `hidden: true` set, this index page is still reachable, it just doesn't show up as a self-referential link within this page and also doesn't show up in the left-hand nav. It can still be linked to. I thought hiding the link would be preferable but I can un-hide this if you'd prefer to keep it.

Reply:
@cjohnhanson ah ok, that's fine for now.